pmlc/conversion/tile_to_pxa/tests/eltwise_add.mlir

```diff
@@ -17,3 +17,43 @@ func @eltwise_add(
 // CHECK: affine.load
 // CHECK: addf
 // CHECK: affine.store
+
+func @eltwise_add_f32_index(%arg0: tensor<4x1xf32>) -> (tensor<4x1xf32>) {
+  %c7 = tile.constant 7
+  %1 = "eltwise.add"(%arg0, %c7) : (tensor<4x1xf32>, index) -> tensor<4x1xf32>
+  return %1 : tensor<4x1xf32>
+}
+
+// CHECK-LABEL: func @eltwise_add_f32_index
+
+func @eltwise_add_f64_index(%arg0: tensor<4x1xf64>) -> (tensor<4x1xf64>) {
+  %c7 = tile.constant 7
+  %1 = "eltwise.add"(%arg0, %c7) : (tensor<4x1xf64>, index) -> tensor<4x1xf64>
+  return %1 : tensor<4x1xf64>
+}
+
+// CHECK -LABEL: func @eltwise_add_f64_index
+
+func @eltwise_add_i32_index(%arg0: tensor<4x1xsi32>) -> (tensor<4x1xsi32>) {
+  %c7 = tile.constant 7
+  %1 = "eltwise.add"(%arg0, %c7) : (tensor<4x1xsi32>, index) -> tensor<4x1xsi32>
+  return %1 : tensor<4x1xsi32>
+}
```

Comment:
Please add more CHECKs to ensure that it's lowering properly.

Reply:
Adde more checks.